tests/baselines/reference/optionalMethods.js

```diff
@@ -142,7 +142,7 @@ declare class Bar {
     e: number;
     a: number;
     b?: number;
-    c?: number | undefined;

```

Comment:
Unsure if this counts as a break or not. the declaration is `c? = 2`. Seems like the `?` is redundant if it can't explicitly be assigned `undefined`, so this is supposed to get an explicit `| undefined` added. I think.

Reply:
Reverted to original behavior.